plugins/org.polarsys.capella.transition.system2subsystem.crossphases/src/org/polarsys/capella/transition/system2subsystem/crossphases/rules/core/PropertyValueGroupRule.java

```diff
@@ -36,6 +38,9 @@ protected EObject getSourceContainer(EObject element_p, EObject result_p, IConte
   protected EObject getBestContainer(EObject element, EObject result, IContext context) {
     if (element.eContainer() instanceof BlockArchitecture) {
       return SystemEngineeringExt.getOwnedSystemAnalysis(SubSystemContextHelper.getTransformedEngineering(context));
+    } else if (element.eContainer() instanceof DataPkg) {

```

Comment:
I think this will always put the PV under the Root architecture.
For instance, it prevent PV propagation under the correct container in cases like:
PhysicalArchitecture > DataPkg > DataPkg > PV

I would put this if in the getDefaultContainer. If (getBlockArchitecture(element) instanceof OA), then return SystemEngineeringExt.getOwnedSystemAnalysis, something like that.